tests/snappi_tests/pfcwd/files/pfcwd_burst_storm_helper.py

```diff
@@ -68,9 +73,10 @@ def run_pfcwd_burst_storm_test(api,
     restore_time_sec = get_pfcwd_restore_time(
         host_ans=duthost, intf=dut_port) / 1000.0
 
-    burst_cycle_sec = poll_interval_sec + detect_time_sec + restore_time_sec + 0.1
-    data_flow_dur_sec = ceil(burst_cycle_sec * BURST_EVENTS)
-    pause_flow_dur_sec = poll_interval_sec * 0.5
+    burst_cycle_sec = 2 * (poll_interval_sec + detect_time_sec + restore_time_sec + 0.1)
```

Comment:
why 2 * ?

Reply:
will change to cisco-8000 only.

Reply:
Original test must be run without changing any test behaviour. Add a multiplier only for the low-traffic scenario.